go/kbfs/libkbfs/prefetcher.go

```diff
@@ -208,6 +212,34 @@ func newBlockPrefetcher(retriever BlockRetriever,
 	return p
 }
 
+func (p *blockPrefetcher) sendOverallSyncStatusLocked() {
+	// Don't send a new status notification if we aren't complete, and
+	// if we have sent one within the last interval.
+	if p.overallSyncStatus.SubtreeBytesFetched !=
+		p.overallSyncStatus.SubtreeBytesTotal &&
+		p.config.Clock().Now().Before(
+			p.lastOverallSyncStatusSent.Add(overallSyncStatusInterval)) {
+		return
+	}
+
+	var status keybase1.FolderSyncStatus
+	status.PrefetchProgress = p.overallSyncStatus.ToProtocolProgress(
+		p.config.Clock())
+	if p.overallSyncStatus.SubtreeBytesTotal ==
+		p.overallSyncStatus.SubtreeBytesFetched ||
+		p.overallSyncStatus.SubtreeBytesTotal == 0 {
+		status.PrefetchStatus = keybase1.PrefetchStatus_COMPLETE
+	} else {
+		status.PrefetchStatus = keybase1.PrefetchStatus_IN_PROGRESS
+	}
```

Comment:
Would it make sense to have a helper function that covers these bytes comparison too, since it's used in [simplefs.go](https://github.com/keybase/client/pull/17118/files#diff-b9b12d1b48b0bfd42d3ebb71a8bf0ffeR2365) as well?

Reply:
Done!